vowel_count_code.py

```diff
@@ -0,0 +1,7 @@
+def vowel_count(string):
+    vowels = set(['a','e','i','o','u','A','E','I','O','U'])
```

Comment:
Great! also not this could have been written:
```suggestion
    vowels = {'a','e','i','o','u','A','E','I','O','U'}
```

Looks like a dictionary except it only has the keys, not the values.  Which is actually what a set is. A dict without values, only keys.